src/os400qc3.c

```diff
@@ -1421,6 +1491,7 @@ pbkdf2(LIBSSH2_SESSION *session, char **dk, const unsigned char *passphrase,
         pkcs5->hashlen;
     if(t > 0xFFFFFFFF)
         return -1;
+    /* FIXME: mac never freed? */
```

Comment:
No, because `alloca()` allocates on stack an thus storage is automatically released at function return, like if it were an automatic local variable.